include/gunrock/graph/csr.hxx

```diff
@@ -20,6 +21,11 @@ namespace graph {
 struct empty_csr_t {};
 
 using namespace memory;
+using vertex_t = int;
+using edge_t = int;
+using weight_t = float;
+using csr_t =
+    gunrock::format::csr_t<memory_space_t::device, vertex_t, edge_t, weight_t>;
```

Comment:
These should not be hard-coded. vertex, edge or weight could be other types as well.

Reply:
The hard-coding is removed.